custom_components/tahoma/sensor.py

```diff
@@ -97,6 +107,14 @@
     "core:FossilEnergyInMWh": f"M{ENERGY_WATT_HOUR}",
     "meters_seconds": SPEED_METERS_PER_SECOND,
 }
+BATTERY_STATES = {
```

Comment:
This is duplicate code, see `tahoma_device.py`.

Reply:
would you rather have?
```py
from .tahoma_device import (
    STATE_BATTERY_FULL,
    STATE_BATTERY_LOW,
    STATE_BATTERY_NORMAL,
    STATE_DEAD,
    TahomaDevice,
)
```